src/relay/backend/contrib/ethosu/compiler_attrs.cc

```diff
@@ -59,6 +60,9 @@ struct EthosUCompilerConfigNode : public tvm::AttrsNode<EthosUCompilerConfigNode
     TVM_ATTR_FIELD(enable_cascader)
         .describe("Whether the cascader should be enabled")
         .set_default(false);
+    TVM_ATTR_FIELD(disable_copying_constants)
+        .describe("Whether copying constants is disabled for case without cascader")
```

Comment:
Perhaps it's worth mentioning here that constants are expected to be in SRAM when this option is enabled?

Reply:
Yes, sure.